view/NFTView.tsx

```diff
@@ -88,13 +94,15 @@ function NFTView() {
       const items = response.data.data?.items;
 
       if (items) {
-        const tt = items.map((ele: any) => ({
-          image: ele.imageUrl,
-          name: ele.name,
-          amount: ele.amount,
-          status: ele.status,
-        }));
-        setList(tt);
+        const refinedCollection = items.map(
+          ({ imageUrl, name, amount, status }: NFTItem) => ({
+            image: imageUrl,
+            name: name,
+            amount: amount,
+            status: status,
```

Comment:
short-hand property 사용하기!

```suggestion
            image: imageUrl,
            name,
            amount,
            status,
```

Reply:
short-hand property!!